webpages/settings/index.js

```diff
@@ -579,26 +793,24 @@ const vue = (window.vue = new Vue({
       }, wait);
     },
     closePickers(e, leaveOpen) {
-      for (let addon of this.$children) {
-        for (let setting of addon.$children) {
-          for (let picker of setting.$children) {
-            if (picker.isOpen && picker.canCloseOutside && e.isTrusted && picker.color && picker !== leaveOpen) {
-              picker.toggle(picker.addon, picker.setting, false);
-            }
-          }
+      (function findPickers(child) {
+        for (const item of child.$children) {
+          findPickers(item);
```

Comment:
Is this a weird recursive function?

Reply:
Either way this must be recursive but I discovered a cool vue thing to make things neater.